GameServerCore/Content/HashFunctions.cs

```diff
@@ -34,7 +37,53 @@ public static uint HashStringSdbm(string section, string name)
 
             return hash;
         }
-    }
 
 
+        // Adapted from http://sanity-free.org/12/crc32_implementation_in_csharp.html
```

Comment:
I thought .NET libraries had CRC32 implemented somewhere in it. Is this required to be taken from another source?

Reply:
We had some discussion about it. There is a lib that doing that and already used somewhere in the code. I'll add this to this project and switch to that lib (https://github.com/force-net/Crc32.NET).